<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="state">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="B1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HELP">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f1Up">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f1Down">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f2Up">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f2Down">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f3Up">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f3Down">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f4Up">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f4Down">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f5Up">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="f5Down">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>83</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FLOOR 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FLOOR 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FLOOR 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FLOOR 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FLOOR 5</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="display">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openDoor">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>open door</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeDoor">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>door close</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
